Gemfile.lock

```diff
@@ -0,0 +1,89 @@
+PATH
+  remote: .
+  specs:
+    loggability (0.15.0.pre20170204094808)
```

Comment:
are you intentionally starting to include the gem lock file?

Reply:
Will remove it.